charter-ietf-plants.md

```diff
@@ -14,11 +14,11 @@ Overhead from post-quantum signatures and keys will add significant costs in two
 
 * Relying parties are presented with signatures from the CA and CT logs. Post-quantum overhead is multiplied per signature, increasing the size and latency of the TLS handshake.
 
-The PLANTS Working Group will define a mechanism that integrates log construction into certificate issuance. This enables batching techniques where one signature can cover multiple key/identifier bindings, e.g. by signing Merkle Tree hashes.
+The PLANTS Working Group will define a mechanism that integrates log construction into certificate issuance. This mechanism will reduce the amount of data that needs to be stored in individual log entries, and will reduce the number of signatures that need to be sent to relying parties in TLS handshake.
```

Comment:
New sentence seems reasonable, but, per comments on other PR and our discussion, losing the "batching techniques" sentence loses what the text was trying to communicate. Without it, the discussion about involving the CA in tree construction doesn't make sense. The request was to integrate the text you wanted to add, without breaking core meaning of the text.

I've gone ahead and done that below:

```suggestion
The PLANTS Working Group will define a mechanism that integrates log construction into certificate issuance, and reduces the amount of data in individual log entries and the TLS handshake. Integrating the log into certificate issuance enables techniques where one signature can cover multiple key/identifier bindings, e.g. by signing Merkle Tree hashes.
```